lib/common/email.sh

```diff
@@ -148,8 +169,13 @@ export -f send_report
 get_uploader_email() {
     local file=$1; shift
     local uploader=`_get_uploader $file`
+
+    log_info "File '$file'"
+    log_info "Uploader '$uploader'"
+
     if [ x"$uploader" != x ]; then
         local uploader_email=`_get_username_email $uploader`
+        log_info "Uploader Email '$uploader_email'"
```

Comment:
It would be way cleaner to have a single log entry for these functions, since they are going to be run constantly and this will make the process.log ridiculously noisy (which it already is!) and harder to find proper log messages. One log line stating "ran this function with these inputs and the result was X".